proto/self_serve/orb/v1/orb.proto

```diff
@@ -2,13 +2,26 @@ syntax = "proto3";
 
 package self_serve.orb.v1;
 
+enum OrbMode {
+  ORB_MODE_UNSPECIFIED = 0;
+  LEGACY = 1;
+  SELF_SERVE = 2;
+}
+
+enum OrbType {
+  ORB_TYPE_UNSPECIFIED = 0;
+  PEARL = 1;
+  DIAMOND = 2;
+}
+
 message NoState {}
 message SelfServeStatus {
   bool is_self_serve_enabled = 1;
 }
 message AnnounceOrbId {
   string orb_id = 1;
```

Comment:
@andronat is it possible to add orb name too? not critical, if not we'll just remove that from UI for now